modules/apps/frontend-js/frontend-js-components-sample-web/src/main/resources/META-INF/resources/js/SampleWalkthrough.js

```diff
@@ -12,47 +12,61 @@
  * details.
  */
 
-import {Walkthrough} from 'frontend-js-components-web';
+import Walkthrough from '@liferay/frontend-js-walkthrough-web/Walkthrough';
```

Comment:
Ideally, we should avoid MODULE_NAME/file. I remember @bryceosterhaus or @kresimir-coko removing those imports a few months ago. Say "go out, Diego" if I'm wrong. 😂

Reply:
Yeah, this was intended to only use in Sample, I didn't want to export in `index.js` because this module is not intended to use the component directly.

Reply:
We have to evaluate potential ways to use the walkthrough from DXP apps. Not for this epic, but just to keep in mind